05-LinqToXml/LinqToXml/LinqToXml.cs

```diff
@@ -59,7 +113,7 @@ public static string GetConcatenationString(string xmlRepresentation)
         /// <returns>Xml representation with contacts (refer to ReplaceCustomersWithContactsResult.xml in Resources)</returns>
         public static string ReplaceAllCustomersWithContacts(string xmlRepresentation)
         {
-            throw new NotImplementedException();
+            return xmlRepresentation.Replace("customer", "contact");
```

Comment:
the solution is fine, but the point of the task is to learn how to modify the tree. Create the tree and modify the elements